recipes/sk-dist/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sk-dist" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fd956610ef88343686c6dcc1e01df617829cdf43444a4d3a41437b214df8a89d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
You can remove this `build` section since it doesn't look like you are compiling any code.